### PR TITLE
UIA in MS Word: don't ignore tables with blank cells

### LINKED ISSUE #8692: MS Word with UIA, table navigation breaks on empty cells
cc @michaeldcurran

### Steps to reproduce:
1. Create a Word document with a table that contains empty cells. [table.docx](https://github.com/nvaccess/nvda/files/2333182/table.docx) is an example that implements a table with 3 columns and two rows. On the first row, the middle cell is empty, whereas in row 2, the right most cell is empty.
2. Navigate with table navigation through all the cells.

### Actual behavior:
NVDA announces out of table when coming across an empty cell, and table navigation does not work any more, unless you manually move back in the table.

### Expected behavior:
It should be possible to access every cell with table navigation.

#### NVDA Installed/portable/running from source:
Installed

#### NVDA version:
NVDA alpha-15933

#### Windows version:
Version 1809 (OS Build 17746.1000). Also reproducible in the April update.

#### Name and version of other software in use when reproducing the issue:
MS Word 16.0.10325.20118

#### Does the issue still occur after restarting your PC?
Yes

#### Have you tried any other versions of NVDA?
Not yet

### PR DESCRIPTION
**Pull request by michaelDCurran**
### Link to issue number:
Closes #8692 

### Summary of the issue:
When navigating in MS Word with NVDA's experimental UI Automation support, blank table cells atre treated as not existing at all.

### Description of how this pull request fixes the issue:
This was fixed in the UIAInMSWordByDefault branch a little while ago. the two relevant commits have been cherry-picked from there.
Those commits are isolated from the rest of the UIAInMSWordByDefault work and are safe to bring in to 2018.4. Stock standard NVDA will never run this code, but for those testing uIA with MS Word, this is very important.
 
### Testing performed:
Arrowed and table navigated around both tables with and with out content in their cells.

### Known issues with pull request:
None.

### Change log entry:
None as the UIA code is still behind a hidden config option.

### CODE REVIEW

File: source/NVDAObjects/UIA/__init__.py [hunk 1]
```diff
@@ -342,8 +342,12 @@ def _get_UIAElementAtStart(self):
 		# Thus we must check getChildren before getEnclosingElement.
 		tempInfo.expand(textInfos.UNIT_CHARACTER)
 		tempRange=tempInfo._rangeObj
-		children=getChildrenWithCacheFromUIATextRange(tempRange,UIAHandler.handler.baseCacheRequest)
-		if children.length==1:
+		try:
+			children=getChildrenWithCacheFromUIATextRange(tempRange,UIAHandler.handler.baseCacheRequest)
```
Comment (leonardder) [inline source/NVDAObjects/UIA/__init__.py]: How is this change related to the table issue?

**Review by LeonarddeR — APPROVED** [review]

**Comment by michaelDCurran** [comment]
The second commit is necessary as the exception it stops did not exist 
without the first commit.

In other words, the overriding of isCollapsed was fixing that exception, 
but caused the tables bug. Rather now, that exception is fixed directly.

**Comment by jage9** [comment]
Thanks, confirming this works in both the sample Doc and other tables I am using in Alpha 16382.
